interface/globals.php

```diff
@@ -192,6 +192,15 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Include the version file
 require_once (dirname(__FILE__) . "/../version.php");
 
+// The logging level for common/logging/logger.php
+$GLOBALS["log_level"] = "DEBUG"; // (value can be TRACE, DEBUG, INFO, WARN, or ERROR)
+
+// Should Doctrine run in "dev mode"?
+$GLOBALS["doctrine_dev_mode"] = false;
+
+// Should Doctrine make use of connection pooling?
+$GLOBALS["doctrine_connection_pooling"] = true;
```

Comment:
This is definitely something that we want to be configured externally. If someone is using a server with limited resources, they will probably want pooling turned off. In fact, I will update the comments around this to make sure that is noted.